src/components/InputText/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './index.css';
+
+
+function TextInput({value,onChange,placeholder,smallInputBox=false}) {
+    return(
+            <input className={smallInputBox? 'smallTextInput': undefined} type='text' value={value} onChange={onChange} placeholder={placeholder}/>
```

Comment:
```suggestion
            <input className={smallInputBox? 'smallTextInput': undefined} type='text' value={value} onChange={onChange} placeholder={placeholder} {...props}/>
```
I will explain this `...props` to you directly, and this line is so long so please make it multi line